src/ui_fsmenu/internet_lobby.h

```diff
@@ -69,8 +69,6 @@ class FullscreenMenuInternetLobby : public FullscreenMenuLoadMapOrGame {
 
 	FullscreenMenuMain& fsmm_;
```

Comment:
You could now get rid of the several local `fsmm_` variables and make this a `protected` variable in  `BaseMenu`